src/main/resources/templates/index.html

```diff
@@ -77,7 +77,7 @@
                         <div th:if="${book.getInventory() > 0}">
                             <form action="/cart/add" method="POST">
                                 <input name="bookId" type="hidden"th:value="${book.getId()}">
-                                <input name="cartId" type="hidden">
+                                <input name="userId" type="hidden">
```

Comment:
Same as above, this functionality uses cookies right?

Reply:
yes it does, also answered above